server/sonar-server/src/main/java/org/sonar/server/authentication/OAuth2ContextFactory.java

```diff
@@ -27,20 +31,16 @@
 import org.sonar.api.server.authentication.UserIdentity;
 import org.sonar.api.utils.MessageException;
 
-import static java.lang.String.format;
-import static org.sonar.api.CoreProperties.SERVER_BASE_URL;
-import static org.sonar.server.authentication.OAuth2CallbackFilter.CALLBACK_PATH;
-
 public class OAuth2ContextFactory {
 
   private final UserIdentityAuthenticator userIdentityAuthenticator;
   private final Server server;
-  private final CsrfVerifier csrfVerifier;
+  private final OAuthCsrfVerifier OAuthCsrfVerifier;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "OAuthCsrfVerifier" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS00116)